package.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "@types/validator",
+  "version": "0.0.1",
```

Comment:
Absolutely! @felixfbecker This version of the types matches `validator 5.7.0`, right?